spec/spec_helper.rb

```diff
@@ -2,6 +2,9 @@
 
 RSpec.configure do |c|
   c.mock_with :rspec
+  c.before(:each) do
```

Comment:
I'm not sure that we want to do this out here. I don't think it will necessarily cause any trouble -- just a question of doing something so global.

Reply:
I meant to have this only in the shared context - thanks